packages/orchestration/src/exos/ibc-packet.js

```diff
@@ -112,6 +123,19 @@ export const prepareIBCTransferSender = (zone, { watch, makeIBCReplyKit }) => {
           const { match } = ctx;
           const { transferMsg } = this.state;
 
+          // sequence + sourceChannel uniquely identifies a transfer.
+          // trace with receiver etc. for forensics
+          const { receiver, sourceChannel } = transferMsg;
+          const { amount } = transferMsg.token;
+          const memoData = tryJSONParse(transferMsg.memo);
+          trace(sourceChannel, '#', sequence, 'to', {
+            receiver,
+            token: { amount },
+            ...(memoData && {
+              forward: { receiver: memoData?.forward?.receiver },
```

Comment:
Is the ? necessary on memoData? I'd expect the && to narrow it.

Should 'receiver` be on `forward`? If so consider a fallback value like 'missing' if it isn't.

Actually, why recreate the 'forward' object instead of passing it through?

I believe we have limits on the size of memo, so why not just include it as `memo` regardless of what's inside?

Reply:
> I believe we have limits on the size of memo ...

@0xpatrickdev suggested that memos can get large and logging them would be awkward in such cases.

logging the whole `transferMsg` is simplest... I dunno... somebody flip a coin or something?

Reply:
ok... logging the whole `transferMsg` looks like this rather than the above in **Testing Considerations**:

```
----- IBCP.6  2 sequence 1 {
  '@type': '/ibc.applications.transfer.v1.MsgTransfer',
  memo: '{"forward":{"channel":"channel-33","port":"transfer","receiver":"dydx1anything","retries":3,"timeout":"10m"}}',
  receiver: 'noble1test1',
  sender: 'agoric1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqp7zqht',
  sourceChannel: 'channel-62',
  sourcePort: 'transfer',
  timeoutHeight: { revisionHeight: 0n, revisionNumber: 0n },
  timeoutTimestamp: 300000000000n,
  token: {
    amount: '149840000',
    denom: 'ibc/FE98AAD68F02F03565E9FA39A5E627946699B2B07115889ED812D8BA639576A9'
  }
}
```